setup.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+DB_USER=postgres
+DB_HOST=127.0.0.1
+DB_PORT=5432
+DB_NAME=$(cat config.json |jq -r .database)
+SPACE=$(cat config.json |jq -r .space)
+API_KEY=$(cat config.json |jq -r .api_key)
+
+PSQL="psql -U ${DB_USER} -h ${DB_HOST} -p ${DB_PORT} -d ${DB_NAME}"
+
+# Create the database
+createdb -h ${DB_HOST} -U ${DB_USER} -p ${DB_PORT} ${DB_NAME}
+
+# Add the extention and metadata tables
+(
+  echo "drop extension if exists multicorn cascade;"
+  echo "create extension multicorn;"
+  echo "create server contentful_space_${SPACE} foreign data wrapper multicorn options ("
+  echo "wrapper 'contentful_fdw.contentful_fdw.ContentfulFDW',"
+  echo "space '"${SPACE}"',"
+  echo "api_key '"${API_KEY}"');"
+  echo "create foreign table content_types ("
+  echo "id varchar,"
+  echo "type varchar,"
+  echo "name varchar,"
+  echo "description varchar,"
+  echo "display_field varchar,"
+  echo "fields jsonb"
+  echo ") server contentful_space_${SPACE} options (type 'ContentType');"
+  echo "create foreign table assets ("
+  echo "id varchar,"
+  echo "type varchar,"
+  echo "title varchar,"
+  echo "description varchar,"
+  echo "file_name varchar,"
+  echo "file_content_type varchar,"
+  echo "file_url varchar,"
+  echo "file_size numeric"
+  echo ") server contentful_space_${SPACE} options (type 'Asset');"
+) | ${PSQL}
```

Comment:
Should we maybe put this into an SQL file? I think this would automagically be syntax highlighted by github :-)